packages/@glimmer/local-debug-flags/index.ts

```diff
@@ -1,5 +1,9 @@
+
+
 export const LOCAL_DEBUG: true | false =
-  import.meta.env.DEV &&
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
I didn't know how to best extend the meta.env type here, so I ignored it for now since it's compiled away during publish, and only used in this one file.